src/App.tsx

```diff
@@ -48,34 +59,62 @@ export function getReorderedGoods(
 // };
 
 export const App: React.FC = () => {
+  const [isReversed, setReversed] = React.useState(false);
+  const [sortType, setSortType] = React.useState(SortType.NONE);
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={`button is-info 
+            ${sortType !== SortType.ALPHABET ? 'is-light' : ''}`}
```

Comment:
pls use classnames library for dynamic classes

Reply:
i am not sure that i get you right. Some googling shows me that first i need to use `npm install classes` and then my code should be like that:
`<button
   type="button"
   className = {classNames('button', 'is-info', {'is-light': sortType !== SortType.LENGTH})}
          onClick={() => setSortType(SortType.LENGTH)}
 >
    Sort by length
 </button>` 
is it correct?

Reply:
exactly!) 👍🏼